demos/flowchart.html

```diff
@@ -1539,7 +1539,11 @@ <h3>flowchart</h3>
       class I bugged_node
     </pre>
     <hr />
-
+    <pre class="mermaid">
+      flowchart LR
+  %% this is a comment A -- text --> B{node}
+  A -- text --> B -- text2 --> C
+    </pre>
```

Comment:
Having this demo here is not really useful. You can remove it. 
If you were using this for development, use the `demos/dev` folder.